### ISSUE TITLE
[feature request] Close tabs to the left

### ISSUE THREAD

**Issue by yannvr** [brave/brave-browser#5194]
Adding a contextual option "Close tabs to the left" would allow the remove all the clutter in one click. That would be simply awesome to navigate efficiently.

**Comment by dagli** [comment]
You can click all the tabs you want to keep while CTRL is pressed, move them left and then do the "close tabs to right" thing :)

**Comment by yannvr** [comment]
yes @dagli, I'm doing that all the time. That's exactly why I think Brave should this feature ;) 
It's very tedious over 20+ tabs (and on high resolution monitor) and I never get it right the first try :(

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
